app/src/main/res/values/strings.xml

```diff
@@ -154,10 +154,11 @@
     <string name="popup_playing_append">Queued on popup player</string>
     <string name="c3s_url" translatable="false">https://www.c3s.cc/</string>
     <string name="content">Content</string>
-    <string name="show_age_restricted_content_title">Age restricted content</string>
-    <string name="video_is_age_restricted">Show age restricted video. Future changes are possible from the settings.</string>
-    <string name="youtube_restricted_mode_enabled_title">YouTube restricted mode</string>
-    <string name="restricted_video">This video is age restricted.\n\nTurn on \"Age restricted content\" in the settings if you want to see it.</string>
+    <string name="show_age_restricted_content_title">Show age restricted content</string>
+    <string name="show_age_restricted_content_summary">Show content with an age limit (like 18+) and thus being probably unsafe for children.</string>
```

Comment:
```suggestion
    <string name="show_age_restricted_content_summary">Show content suitable for ages 18 and above.</string>
```

Reply:
All content is **suitable** for ages 18 and above (i.e. the statement is ill-formed, as it is always true); Newpipe also restricts content 14+, 12+, 6+, ... (you name it, even though no service has such specific labels so far afaik); I wanted to talk about children safety just to be more clear, but this can indeed be removed.

Reply:
Wat really? So if a video is marked 6+ Newpipe would block it if the setting was off?

Added "only".

Reply:
What about "show content only suitable..."

Reply:
It would mean we show only 18+ content, not anything else. :P

Reply:
How about just directly saying "Show NSFW content"?

Reply:
Ooh. We could diss Youtube a bit.

"Show content Youtube deems unsafe for children below 18".

Reply:
This is not only for Youtube and NSFW does not imply age restricted. Also, one would have to go look for NSFW on urban dictionary (like me a while ago). I'd keep your current suggestion then, it is clear what it means imo

Reply:
Would be people below 18. Isn't the limit used to bury content too?

Reply:
```suggestion
    <string name="show_age_restricted_content_summary">Show content intended for a mature audience.</string>
```

Reply:
This is flawed for two reasons
1. It suggests the material will be adjusted for the mental capacities of someone deemed to be at a mature level.
2. It suggests the same could be acquired by cutting material intended for immature audiences (it isn't(?)
3. By turning it off (I imagine it is on by default), the same happens. Is now content for mature audiences cut?

Intention is not a good guide for what actually happens, regardless of what actually happens.

Reply:
I was going by the idea that this sort of terminology is already used when classifying content for various audiences.

ESRB has a Mature category, for example. And I'm sure you must have seen the quote "The following is intended only for mature audiences. Viewer discretion advised. " several times.